exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -167,10 +167,12 @@ drill.exec: {
     debug: true,
     janino_maxsize: 262144,
     cache_max_size: 1000,
-    // Enable to write generated source to disk. See ClassBuilder
-    save_source: false,
     // Where to save the generated source. See ClassBuilder
     code_dir: "/tmp/drill/codegen"
```

Comment:
Using Eclipse, the /tmp/drill/codegen directory is very simple: Eclipse prompts for the source location on the first run, remembers it thereafter, and MacOS cleans up any left-over generated files on the next restart.

Looks like things are a bit more difficult in IntelliJ. I'll play around a bit to see if I can find a solution that works well for both tools. I'll create a separate JIRA and PR for that work.